scripts/rollup/shims/react-native-fb/ReactFeatureFlags.js

```diff
@@ -14,6 +14,7 @@ const ReactFeatureFlags = {
   debugRenderPhaseSideEffects: false,
   debugRenderPhaseSideEffectsForStrictMode: false,
   warnAboutDeprecatedLifecycles: true,
+  warnAboutLegacyContextAPI: true,
```

Comment:
Curiosity question: Did we intentionally turn this on for RN renderer but not but for www? (It's probably good to turn it on for both.)

Reply:
I'm not sure about this. Maybe ask @acdlite 😄

Reply:
Let's turn it on for both.

Reply:
If we realize this causes problems during the next www sync (which I'll be running) we can back it off then.

Reply:
Why are they ever off? It's only read in strict mode, yeah? My understanding was that these `warnAbout-` feature flags only exist so we can turn them off in certain tests.

Reply:
The two side effect flags are controlled by GKs internally. They're off by default, but can be turned on by injection. (xplat feature flags can't directly reference `MobileConfig` for GKs like www can.)